ci/tests/verify_wasm_assets.py

```
@@ -0,0 +1,164 @@
+        # text/plain -- and a module script served as text/plain is rejected
+        # outright by strict MIME checking, so nothing loads. Pin the types
+        # that matter rather than depending on the host's file associations.
+        extensions_map = {
+            **http.server.SimpleHTTPRequestHandler.extensions_map,
+            ".js": "text/javascript",
+            ".mjs": "text/javascript",
+            ".wasm": "application/wasm",
+            ".json": "application/json",
+            ".css": "text/css",
+            ".map": "application/json",
+        }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)